k8s/workloads/careers/careers-ingress-prod.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ingress-nginx
+  namespace: careers-prod
+  labels:
+    app: careers
+spec:
+  releaseName: careers-ingress-nginx
+  chart:
+    repository: https://kubernetes.github.io/ingress-nginx
+    name: ingress-nginx
+    version: 3.23.0
+  values:
+    controller:
+      useIngressClassOnly: true
+      enableCustomResources: false
+      autoscaling:
+        enabled: true
+        minReplicas: 1
+        maxReplicas: 4
+        targetCPUUtilizationPercentage: 80
+        targetMemoryUtilizationPercentage: 80
+      resources:
+        requests:
+          cpu: 100m
+          memory: 128Mi
+        limits:
+          cpu: 200m
+          memory: 256Mi
+      admissionWebhooks:
+        enable: false
+      scope:
+        enabled: true
+      config:
+        enable-vts-status: true
+        force-ssl-redirect: true
+        # Rate limitting: defaults x7
+        limit-connections: "140"
+        limit-rpm: "1400"
+        limit-rps: "84"
+        proxy-body-size: 4m
+        proxy-buffer-size: 16k
+        # restrict this to the IP addresses of ELB
+        proxy-real-ip-cidr: "0.0.0.0/0"
+        ssl-redirect: true
+        use-forwarded-headers: "true"
+        use-proxy-protocol: "false"
+      service:
+        externalTrafficPolicy: "Local"
+        annotations:
+          service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
+          service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: "true"
+          service.beta.kubernetes.io/aws-load-balancer-connection-draining-enabled: "true"
+          service.beta.kubernetes.io/aws-load-balancer-additional-resource-tags: "Environment=prod"
+          external-dns.alpha.kubernetes.io/hostname: "careers.prod.mozit.cloud"
+    rbac:
```

Comment:
Is this used? Trying to find `rbac` over here https://github.com/mozilla-it/helm-charts/blob/main/charts/careers/templates/ingress.yaml

Reply:
won't be in my chart, this is the deployment of the nginx controller. (we found managing this separately kept us from thrashing the nlb as often, basically the nlb will be deployed once per environment rather than once per application deployment). - https://kubernetes.github.io/ingress-nginx